src/netstandard/ref/System.Security.Cryptography.cs

```diff
@@ -10,6 +10,18 @@ public abstract partial class Aes : System.Security.Cryptography.SymmetricAlgori
         public static new System.Security.Cryptography.Aes Create() { throw null; }
         public static new System.Security.Cryptography.Aes Create(string algorithmName) { throw null; }
     }
+    public sealed partial class AesCcm : System.IDisposable
+    {
+        public AesCcm(byte[] key) { }
+        public AesCcm(System.ReadOnlySpan<byte> key) { }
+        public static System.Security.Cryptography.KeySizes NonceByteSizes { get { throw null; } }
+        public static System.Security.Cryptography.KeySizes TagByteSizes { get { throw null; } }
+        public void Decrypt(byte[] nonce, byte[] ciphertext, byte[] tag, byte[] plaintext, byte[] associatedData = null) { }
```

Comment:
The lower-case "t" in `ciphertext` and `plaintext` bugs me (and is inconsistent with the choice made in the sample in https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.aes for example) but I assume that can't be changed now. (Even if it could, I anticipate it would cause debates...)

Reply:
FWIW, both `plaintext` and `ciphertext` are single words in the Merriam-Webster dictionary: https://www.merriam-webster.com/dictionary/plaintext and https://www.merriam-webster.com/dictionary/ciphertext.  (Which, of course, can then progress to the "but it used to be a compound word" debate... I'm just offering evidence that it isn't strictly wrong)

Reply:
Unless there are substantial concerns I'm inclined to leave the API as-is. Changing that now would be a (albeit source breaking change) for .NET Core. Not sure that's worth it.

Reply:
@jskeet what do you think?

Reply:
Never mind. @bartonjs just showed me how GitHub works ;-)